docs/guides/theming/Theming.stories.mdx

```diff
@@ -363,6 +365,67 @@ const MyComponent = () => {
   <CssVariables />
 </Story>
 
+## Default Props <a id="default-props" />
+
+Through our theme it is also possible to change the default props of a component. Let's say for example that the theme you want only uses squared
+shape avatars, you can use this feature to make your life easier by pre defining the default value of the prop `variant` to `square` in the `theme` `components` section like so:
+
+```tsx
+const newTheme = createTheme({
+  name: "myTheme",
+  base: "ds5",
+  components: {
+    HvAvatar: {
+      variant: "square",
+    },
+  },
+});
+
+export const MyApp = () => {
+  return (
+    <HvProvider themes={[newTheme]}>
+      <HvAvatar />
+    </HvProvider>
+  );
+};
+```
+
+<Story name="Default Props">
+  <DefaultProps />
+</Story>
+
+This allows us to not have to declare that property every time we use the component.
+One of the big benefits of this is that most of all of our components have a `classes` prop. This means that we can now customize all
+of the components by overwriting their classes through the theme object.
+
+```tsx
+const newThemeClassOverwrite = createTheme({
```

Comment:
It should be `newTheme` and not `newThemeClassOverwrite` as defined in the `HvProvider` below.

Also, could we add the imports just for the sample to be similar to the other ones (and also users may find the imports useful)?

```
import { createTheme, HvProvider, HvButton } from "@hitachivantara/uikit-react-core";
```